### PR TITLE
Implementation of the Profile-Payment UI

### PR DESCRIPTION
**Pull request by BerryGIT-ME**
### What this PR does (required):
- Adds the UI for managing payment cards 

### Screenshots / Videos (required):
Payment UI

payment UI input

### Any information needed to test this feature (required):
Please see the integration PR for tests [here](https://github.com/hatchways/team-brioche/pull/83#issue-1020371733)

### CODE REVIEW

File: client/src/helpers/APICalls/paymentService.ts [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+
+interface AddPaymentResponse {
+  clientSecret: string;
+  attachedDetails: {
+    name: string;
+    email: string;
+  };
+}
+export const createCardSetup = async (): Promise<AddPaymentResponse> => {
+  // create setup intent
```
Comment (ezeilo-su) [inline client/src/helpers/APICalls/paymentService.ts:11]: Remove comment

File: client/src/helpers/APICalls/paymentService.ts [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+  paymentMethods: PaymentMethod[];
+  defaultPaymentMethod: string;
+}
+export const getAllPaymentMethodsByCustomer = async (): Promise<GetPaymentMethodsResponse> => {
+  return { paymentMethods: [], defaultPaymentMethod: '' };
+};
+
+export const updateDefaultMethod = async (methodId: string): Promise<void> => {
+  // No error means ok
+};
```
Comment (ezeilo-su) [inline client/src/helpers/APICalls/paymentService.ts:25]: No function body?

**Review by ezeilo-su — COMMENTED** [review]
Your UI is beautiful :+1:  I left a comment for you to take a look at.
Happy coding!

File: client/src/components/ProfilePayment/ProfilePayment.tsx [hunk 3]
```diff
@@ -0,0 +1,164 @@
...
+  getAllPaymentMethodsByCustomer,
+  updateDefaultMethod,
+} from '../../helpers/APICalls/paymentService';
+import withStripe from './withStripe';
+import useStyles from './useStyles';
+import visa from '../../Images/cardImages/Visa.png';
+import mastercard from '../../Images/cardImages/mastercard.jpg';
+import defaultCardLogo from '../../Images/cardImages/default.png';
+
+const cardLogos = [
```
Comment (rajivtitus) [inline client/src/components/ProfilePayment/ProfilePayment.tsx:20]: 👍🏽

File: client/src/helpers/APICalls/paymentService.ts (hunk 1 above)
Comment (rajivtitus) [reply client/src/helpers/APICalls/paymentService.ts:11]: @BerryGIT-ME you can actually remove the above comment

File: client/src/helpers/APICalls/paymentService.ts (hunk 2 above)
Comment (rajivtitus) [reply client/src/helpers/APICalls/paymentService.ts:25]: @BerryGIT-ME In the future, consider adding a small TODO comment where applicable :)

**Pull request merged**